meta-ov/recipes-apps/ovmenu-ng-skripts/files/upload-xcsoar.sh

```diff
@@ -2,15 +2,11 @@
 
 USB_PATH="/usb/usbstick/openvario/upload/xcsoar"
 UPLOAD_PATH="/home/root/.xcsoar"
-if [ -z "$(ls $USB_PATH/* 2>/dev/null)" ]; then
+if [ -z "$(find "$USB_PATH" -type f 2>/dev/null)" ]; then
         echo "No files found !!!"
 else
-        for uploadfile in $(find $USB_PATH -type f); do
-                echo $uploadfile
-                cp $uploadfile $UPLOAD_PATH
-        done
+        find "$USB_PATH" -type f -exec basename {} \; -exec cp "{}" "$UPLOAD_PATH" \;
 fi
 echo "Umount Stick ..."
-sync
```

Comment:
Why remove the `sync`?

Reply:
Because of the unmount, the sync should be automatic - but my question is: Why is the unmount happening here? The shell script does not mount the USB stick, with subsequent actions (e.g. download-all.sh) the stick is unexpectedly not mounted! The same applies to all other scripts!
From my point of view, the 'unmount' should be its own menu item!

Reply:
> Because of the unmount, the sync should be automatic

Only sync to the to-be-unmounted filesystem is implicit. Which is not relevant here.

If removing a `sync` command is useful, that fact should be documented, and should not be hidden inside a unrelated commit. That makes patch review harder and can lead to unintended regressions.

> - but my question is: Why is the unmount happening here? The shell script does not mount the USB stick

That is a good question, and I was wondering the same - but since this is old code which is left untouched in this PR, I didn't ask it here.

Reply:
@MaxKellermann: Are you using the scripts in your new OV menu in XCSoar? Or is the effort here obsolete?
In addition, we also have Kedders (Python) menu...

Reply:
Yes, so cleanup here is good - even if they get ported to C++ eventually, the lessons learned from existing code is always useful.

Reply:
I re added the sync in my new version, but removed the umount. This worked before, as it was immediately remounted by autofs.